README.md

```diff
@@ -19,20 +19,36 @@ or check out examples below.
 # In your config/config.exs file
 config :my_app, Repo,
   adapter: Mongo.Ecto,
+  # Example key-value configuration:
   database: "ecto_simple",
   username: "mongodb",
-  password: "mongosb",
+  password: "mongodb",
   hostname: "localhost"
+  # OR you can configure with a connection string (mongodb:// or mongodb+srv://):
+  mongo_url: "mongodb://mongodb:mongodb@localhost:27017/ecto_simple"
+
+config :my_app,
+  # Add Repo to this list so you can run commands like `mix ecto.create`.
+  ecto_repos: [Repo]
 
 # In your application code
 defmodule Repo do
-  use Ecto.Repo, otp_app: :my_app
+  use Ecto.Repo,
+    otp_app: :my_app,
+    adapter: Mongo.Ecto
+
+  def pool() do
+    Ecto.Adapter.lookup_meta(__MODULE__).pid
```

Comment:
@mweidner037 In our code base, we do `%{pid: pid} = Ecto.Repo.Registry.lookup(__MODULE__)`. Should we switch to this form or are they equivalent?

Reply:
They appear to be equivalent, though perhaps the `Ecto.Adapter.lookup_meta` form is preferable since that function is actually documented. I'll update it when changing the Mongo Ecto versions.